time stamp
string uav_name
int32 priority
bool collision_avoidance
float32 min_radius 
